### PR TITLE
Add cell for Vtune link

### PR DESCRIPTION
**Pull request by gartung**

### CODE REVIEW

**Comment by gartung** [comment]
@smuzaffar

File: report-summary-merged-prs.py [hunk 1]
```diff
@@ -1900,6 +1907,13 @@ class PossibleUnitTestResults(object):
                         MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER3,
                         find_one_profiling_result,
                     )
+                    find_general_test_results(
+                        "vtune",
+                        release_queue_results["comparisons"],
+                        arch,
+                        MAGIC_COMMAND_FIND_VTUNE_CHECKS_FILTER,
+                        find_one_vtune_result,
```
Comment (smuzaffar) [inline report-summary-merged-prs.py:1915]: @gartung , I do not see `find_one_vtune_result` function?

File: templates/js/render.js [hunk 2]
```diff
@@ -436,6 +436,25 @@ add_igprof_tests_link = function ( title_cell, isFound, currentTag ){
...
+  if ( isFound == 'found' ){
+    var url = CMSSDT_SERVER+'/vtune/ui/' + currentTag
+    var sa_link = $("<a></a>").attr("href", url)
+    sa_link.append($('<span class="glyphicon glyphicon-list-alt"></span>'))
+    sa_link.append($('<span></span>').text(' Vtune'))
+    title_cell.append(sa_link)
+    title_cell.append($("<br>"))
+  }
+}
+
```
Comment (smuzaffar) [inline templates/js/render.js:457]: @gartung , I think this was for old IB page which we do not have any more. So change in this file is not needed

File: report-summary-merged-prs.py (hunk 1 above)
Comment (gartung) [reply report-summary-merged-prs.py:1930]: added

File: templates/js/render.js (hunk 2 above)
Comment (gartung) [reply templates/js/render.js]: reverted change

**Comment by smuzaffar** [comment]
looks good though https://github.com/cms-sw/cms-bot/pull/2223/files#diff-764d6cd575512ae10e52cad8742ae423500083ad0646b03719342a37f40dc91bR1072 line is not needed as there is no `WORKFLOW` keyword in the magic command

**Pull request merged**